src/app-layout/visual-refresh-toolbar/skeleton/styles.scss

```diff
@@ -139,7 +139,10 @@
   z-index: 840;
   align-self: end;
   grid-area: main;
-  @include grid-column-full-content-width;
+  grid-column: 1 / -1;
+  @include desktop-only {
+    grid-column: 2 / span 4;
```

Comment:
Why does this one require a special state?

I checked on [the preview page](https://d21d5uik3ws71m.cloudfront.net/components/5304f73ef6208cb82e29e4eb2c74a65c1404ad3c/dev-pages/index.html#/light/app-layout/disable-paddings-with-split-panel/?appLayoutToolbar=true&splitPanelOpen=true), found no issues when overriding this to `grid-column: 2 / 5;`

Reply:
You are right, addressed!